mode-line.lisp

```diff
@@ -259,7 +259,8 @@ timer.")
 (defun destroy-mode-line (ml)
   (run-hook-with-args *destroy-mode-line-hook* ml)
   (xlib:destroy-window (mode-line-window ml))
-  (xlib:free-gcontext (mode-line-gc ml))
+  (unless (mode-line-cc ml) ;; mode-lines of dock-type windows do not have a cc
```

Comment:
@m-g-r Should this be `when`?

Reply:
Yes, of course, sorry! @PhilipBAdams noted as well: https://github.com/stumpwm/stumpwm/issues/777#issuecomment-625352294

As I cannot reopen this, I've made a new PR: